packages/web/src/components/usdc-purchase-details-modal/components/SaleModalContent.tsx

```diff
@@ -71,7 +84,91 @@ export const SaleModalContent = ({
     dispatch
   ])
 
-  return (
+  return isPremiumAlbumsEnabled ? (
+    <>
+      <ModalHeader>
+        <ModalTitle title={messages.saleDetails} />
+      </ModalHeader>
+      <ModalContent className={styles.content}>
+        <Flex gap='xl' direction='column' w='100%'>
+          <Flex
+            borderBottom='default'
+            gap='l'
+            justifyContent='spaceBetween'
+            w='100%'
+            pb='xl'
+          >
+            <DynamicTrackArtwork
+              id={purchaseDetails.contentId}
+              size={DynamicTrackArtworkSize.LARGE}
+            />
+            <DetailSection label={messages.track}>
+              <TrackLink onClick={onClose} id={purchaseDetails.contentId} />
+              <Flex gap='xs'>
+                <Text variant='body' size='l' textTransform='lowercase'>
+                  {messages.by}
+                </Text>
+                <Text variant='body' size='l' color='accent'>
+                  <UserNameAndBadges
+                    onNavigateAway={onClose}
+                    userId={purchaseDetails.sellerUserId}
+                  />
+                </Text>
+              </Flex>
+            </DetailSection>
+          </Flex>
+          <Flex
+            justifyContent='space-between'
+            w='100%'
+            borderBottom='default'
+            pb='xl'
+          >
+            <Flex gap='s' direction='column'>
+              <Text variant='label'> {messages.purchasedBy}</Text>
```

Comment:
space?

Reply:
basically same callouts in last modal for this one too :)

Reply:
hm thought prettier would've caught the space